doc/submitter_guide.md

```diff
@@ -60,12 +60,11 @@ dashboard](https://kcidb.kernelci.org/).
 2\. Install KCIDB
 -----------------
 
-Pick a KCIDB [release][releases]. Latest would normally be best.
 
-Let's say you picked v9. Run this:
+KCIDB employs continuous integration and delivery, and aims to keep the code working at all times. Please install the latest version from GitHub:
```

Comment:
Please wrap the text at the 78 character boundary. This makes the raw documentation easier to read (and Markdown is intended to be read raw as well).

Reply:
okay